htdocs/commande/card.php

```diff
@@ -112,6 +112,10 @@
 $permissiondellink = $usercancreate; 	// Used by the include of actions_dellink.inc.php
 $permissiontoadd = $usercancreate; 		// Used by the include of actions_addupdatedelete.inc.php and actions_lineupdown.inc.php
 
+if (!empty($conf->expedition->enabled)){
+	if (empty($object->warehouse_id) && !empty($conf->global->MAIN_DEFAULT_WAREHOUSE)) $object->warehouse_id = $conf->global->MAIN_DEFAULT_WAREHOUSE;
```

Comment:
Property warehouse_id exists only when $conf->global->WAREHOUSE_ASK_WAREHOUSE_DURING_ORDER is defined.
So we must still have a condition on this to enter this code.